tests/custom_maven_install/m2local_testing_with_pinned_file_install.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
How come this lock file is in the old format and not the new one? Could you please regenerate it using the current `HEAD` of `master`?

Reply:
My mistake, was working off an older commit. Regenerated the file now.